### DISCUSSION TITLE
Update your systems to 1.0.2 (For 1.0 and 1.0.1)

### DISCUSSION THREAD

**Discussion by dat-linux** [dat-linux/community#12] [Announcements]
# 1.0 users:
sudo sed -i 's/control-panel-1.0-lxde/control-panel-1.0.x-lxqt/g' /etc/apt/sources.list.d/datlinux.list
# 1.0.1 users:
sudo sed -i 's/control-panel-1.0.1-lxqt/control-panel-1.0.x-lxqt/g' /etc/apt/sources.list.d/datlinux.list
# All:
sudo apt update && sudo apt install --reinstall datlinux-ctrl-pnl

Any issues please reply here.

**Comment by mdwarf** [comment]
Hi, I installed from scratch 1.0.2 version, inside a VM Ware Workstation PRO 16 environment: after completing installation, I launch Dat Linux distribution, boot is ok, but when I open a terminal window and try to switch to root user (su - or sudo followed by a command) I always obtain "si: Athentication failure" or password not valid from sudo commands. I deleted first installation and did it again choosing a very simple password but the problem remains. In the first installation I selected italian locale, in the second one english locale, but nothing changes. What am I mistaking? Thank you.

**Reply by dat-linux** [reply]
Hi, you don;t "switch" to root user in Ubuntu. Just run commands starting with "sudo" if you need to run a command as root. The password is your user password. There is no root password.

https://phoenixnap.com/kb/linux-sudo-command

**Comment by MarcosQuicimbo** [comment]
Hello, I was trying to download an .iso file from the official site [1](https://sourceforge.net/projects/dat-linux/) but it doesn't work

**Reply by dat-linux** [reply]
https://sourceforge.net/projects/dat-linux/files/
